mautic/mautic
# Remove transChoice() as it is deprecated in Symfony 4.2

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | ❌ 
| New feature/enhancement? (use the a.x branch)      | ✅ 
| Deprecations?                          | ❌ 
| BC breaks? (use the c.x branch)        | ❌ 
| Automated tests included?              | ✅  
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes https://mautic.atlassian.net/browse/TPROD-286 



#### Description:
The transchoice tag is deprecated since Symfony 4.2 and will be removed in 5.0. Use the trans with '%count%' parameter instead.



#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Go to UI pages where translation's plural forms are shown for e.g. segment, tag list pages




<a href="https://gitpod.io/#https://github.com/mautic/mautic/pull/11046"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
